src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -255,23 +277,52 @@ public static Map<String, Object> getPersonalisationForAcknowledgement(CaseDetai
         );
     }
 
-    private static TseReplyData createDataForTseReply(CaseData caseData, GenericTseApplicationType application) {
+    private static TseDecisionData createDataForTseDecision(CaseData caseData) {
 
-        return TseReplyData.builder()
+        return TseDecisionData.builder()
             .caseNumber(defaultIfEmpty(caseData.getEthosCaseReference(), null))
-            .respondentParty(RESPONDENT_TITLE)
-            .type(defaultIfEmpty(application.getType(), null))
-            .responseDate(UtilHelper.formatCurrentDate(LocalDate.now()))
-            .response(defaultIfEmpty(application.getDetails(), null))
-            .supportingYesNo(hasSupportingDocs(caseData.getTseResponseSupportingMaterial()))
-            .documentCollection(getUploadedDocList(caseData))
-            .copy(defaultIfEmpty(application.getCopyToOtherPartyYesOrNo(), null))
+            .notificationTitle(caseData.getTseAdminEnterNotificationTitle())
+            .decision(caseData.getTseAdminDecision())
+            .decisionDetails(caseData.getTseAdminDecisionDetails())
+            .typeOfDecision(caseData.getTseAdminTypeOfDecision())
+            .responseRequired(YES.equals(caseData.getTseAdminIsResponseRequired())
+                    ? TSE_ADMIN_DECISION_RESPONSE_REQUIRED_MESSAGE : NO)
+            .selectedRespondents(caseData.getTseAdminSelectPartyRespond())
+            .additionalInformation(caseData.getTseAdminAdditionalInformation())
+            .supportingMaterial(getSupportedDocumentsList(caseData))
+            .decisionMadeBy(caseData.getTseAdminDecisionMadeBy())
+            .decisionMakerFullName(caseData.getTseAdminDecisionMadeByFullName())
+            .notificationParties(caseData.getTseAdminSelectPartyNotify())
```

Comment:
Adding defaultIfEmpty to avoid NPEs

```suggestion
                .notificationTitle(defaultIfEmpty(caseData.getTseAdminEnterNotificationTitle(), ""))
                .decision(defaultIfEmpty(caseData.getTseAdminDecision(), "")
                .decisionDetails(defaultIfEmpty(caseData.getTseAdminDecisionDetails(), ""))
                .typeOfDecision(defaultIfEmpty(caseData.getTseAdminTypeOfDecision(), ""))
                .responseRequired(YES.equals(caseData.getTseAdminIsResponseRequired())
                        ? TSE_ADMIN_DECISION_RESPONSE_REQUIRED_MESSAGE : NO)
                .selectedRespondents(defaultIfEmpty(caseData.getTseAdminSelectPartyRespond(), ""))
                .additionalInformation(defaultIfEmpty(caseData.getTseAdminAdditionalInformation(), ""))
                .supportingMaterial(getSupportedDocumentsList(caseData))
                .decisionMadeBy(defaultIfEmpty(caseData.getTseAdminDecisionMadeBy(), ""))
                .decisionMakerFullName(defaultIfEmpty(caseData.getTseAdminDecisionMadeByFullName(), ""))
                .notificationParties(defaultIfEmpty(caseData.getTseAdminSelectPartyNotify(), ""))
```